### PR TITLE
Assignment of Tool tip text to the clear button in the search palette bar

### PR DESCRIPTION
**Pull request by the-next-ceo**
Resolves: *(direct link to the issue)*

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by the-next-ceo** [comment]
https://user-images.githubusercontent.com/66167731/163622477-a8062735-3124-40d5-a83b-a458574211f1.mov

**Comment by HemantAntony** [comment]
Also, you might want to squash the two commits to have a single commit and finally you would have to resolve conflicts

**Comment by the-next-ceo** [comment]
> Also, you might want to squash the two commits to have a single commit and finally you would have to resolve conflicts. If you have any questions, please ask

do I have to make a pull request again, after squashing two commits??

**Comment by HemantAntony** [comment]
> do I have to make a pull request again, after squashing two commits??

I am assuming you have squashed the commits and you are using terminal. Then open terminal and type this in `git push -f origin branchName`

I don't know for sure if this is the right command (I don't use terminal anymore). So tell me if you get errors

**Comment by the-next-ceo** [comment]
there's a problem it got squashed with the previous commit, not with the latest commit.  do I need to use fixup for that?

**Comment by HemantAntony** [comment]
I think it worked. There's only one commit. It contains the changes you made

Now, you should remove and add the lines you added and removed unnecessarily. I left you reviews to help you out

**Comment by the-next-ceo** [comment]
can u accept my request on discord? the problem does exist there

**Comment by Jojo-Schmitz** [comment]
Apparently a rebase is needed. `git pull --rebase upstream master`, then `git rebase -i HEAD~3`, repley ther `push` on the 2nd and 3rd commit with an f, save, `git push -f`

**Comment by Jojo-Schmitz** [comment]
BTW: it isn't wise to work and recate PRs on top of your master branch, better create a topic branch. Too late not, but for the future

File: src/palette/qml/MuseScore/Palette/internal/PalettesPanelHeader.qml [hunk 1]
```diff
@@ -116,7 +116,7 @@ Item {
 
         navigation.panel: navPanel
         navigation.order: 1
-
+
```
Comment (Jojo-Schmitz) [inline src/palette/qml/MuseScore/Palette/internal/PalettesPanelHeader.qml:119]: whitespace damage

File: src/palette/qml/MuseScore/Palette/internal/PalettesPanelHeader.qml [hunk 2]
```diff
@@ -130,7 +130,7 @@ Item {
             paletteProvider: root.paletteProvider
 
             navigationParentControl: addPalettesButton.navigation
-
+
```
Comment (Jojo-Schmitz) [inline src/palette/qml/MuseScore/Palette/internal/PalettesPanelHeader.qml:133]: same

**Comment by HemantAntony** [comment]
He is currently working on deleting the unnecessary changes (I am helping him)

File: src/palette/qml/MuseScore/Palette/internal/PalettesPanelHeader.qml (hunk 2 above)
Comment (Jojo-Schmitz) [reply src/palette/qml/MuseScore/Palette/internal/PalettesPanelHeader.qml]: OK, this one it gone now

**Comment by Jojo-Schmitz** [comment]
Now it'd be `git rebase -i HEAD~5`

**Comment by the-next-ceo** [comment]
@bkunda I assigned tooltip text to the cross button in the search palette field as **Clear**, as you can see in the video.

**Comment by bkunda** [comment]
> @bkunda I assigned tooltip text to the cross button in the search palette field as **Clear**, as you can see in the video.

This makes sense to me. Thanks @the-next-ceo!

**Comment by the-next-ceo** [comment]
how to remove the latest commit from this pull request? i don't know why it got merged with this one.

**Comment by HemantAntony** [comment]
Almost the same as squashing. You have to do `git rebase -i HEAD~2`. Then move over to the commit you want to remove. Replace the 'pick' with 'drop'. Then save and force push

**Comment by the-next-ceo** [comment]
> Almost the same as squashing. You have to do `git rebase -i HEAD~2`. Then move over to the commit you want to remove. Replace the 'pick' with 'drop'. Then save and force push

https://stackoverflow.com/a/66020341
Will this serve my purpose? I don't want the commit to be deleted from my forked repo, because I think Drop will remove the commit from both sides. Correct me if I'm wrong

**Comment by cbjeukendrup** [comment]
@the-next-ceo Both ways will apply both to your local clone and to your fork on GitHub. You will want to move the second commit to a new branch:
```sh
git checkout master 
git checkout -b lyrics # replace "lyrics" with whatever name you like
# Now you are on a new branch based on master. Cherry-pick the commit to this branch:
git cherry-pick 5156aa0 # the sha of your commit
# Now you're commit lives on the "lyrics" branch. You can now safely delete it from any other branch.
```

**Comment by HemantAntony** [comment]
> @the-next-ceo Both ways will apply both to your local clone and to your fork on GitHub. You will want to move the second commit to a new branch:
> 
> ```shell
> git checkout master 
> git checkout -b lyrics # replace "lyrics" with whatever name you like
> # Now you are on a new branch based on master. Cherry-pick the commit to this branch:
> git cherry-pick 5156aa0 # the sha of your commit
> # Now your commit lives on the "lyrics" branch. You can now safely delete it from any other branch.
> ```

So that's what cherry-pick does. I have heard of it though

**Comment by Eism** [comment]
duplicate of #11314

**Pull request closed without merging**